src/0_storage.js

```diff
@@ -17,7 +19,8 @@ var storage;
 
 if (TITANIUM_BUILD) {
 	/** @typedef {{listProperties: function(), setString: function({string}, {string}), getString: function({string})}}*/
-	Ti.App.Properties;
+	// todo(dmitri): what does this line do? do you actuall mean `prop = Ti.App.Properties;`?
```

Comment:
This looks like a no-opt, but would prefer keeping todo's out of the public SDK. Can you a separate issue to get clarity on this. Can probably confirm with Scott today too.